src/ground_script_1.c

```diff
@@ -0,0 +1,1863 @@
+    u8 kind;
+    u8 unk1;
+    u8 widthTiles;
+    u8 heightTiles;
+    PosInfo pos;
+    ScriptCommand *scripts[4]; // 2 - dialogue script
+} GroundObjectData;
+typedef struct GroundEffectData {
+    u8 kind;
+    u8 unk1;
+    u8 widthTiles; //???
+    u8 heightTiles;
+    PosInfo pos;
+    ScriptCommand *script;
+} GroundEffectData;
+
+void GroundSprite_Reset(s16);
+void GroundMap_Select(s16);
+void GroundMap_SelectDungeon(s16, DungeonLocation*, u8);
+void GroundMap_ExecuteEnter(s16);
+void GroundMap_ExecuteEvent(s16, bool8);
+void GroundMap_GetStationScript(ScriptInfoSmall *out, s16, s32, s32);
+void GroundMap_ExecuteStation(s32, s32, s32, bool8);
+void GroundLives_ExecuteScript(s32, s16 *, ScriptInfoSmall *);
+void GroundObject_ExecuteScript(s32, s16 *, ScriptInfoSmall *);
+void GroundEffect_ExecuteScript(s32, s16 *, ScriptInfoSmall *);
+void GroundLink_Select(s16);
+void GroundLives_Select(s16, s32 group, s32 sector);
+void GroundObject_Select(s16, s32 group, s32 sector);
+void GroundEffect_Select(s16, s32 group, s32 sector);
+void GroundEvent_Select(s16, s32 group, s32 sector);
+void GroundLives_Cancel(s32 group, s32 sector);
+void GroundObject_Cancel(s32 group, s32 sector);
+void GroundEffect_Cancel(s32 group, s32 sector);
+void GroundEvent_Cancel(s32 group, s32 sector);
+void GroundLives_CancelBlank_1(void);
+void GroundObject_CancelBlank(void);
+void GroundEffect_CancelBlank(void);
+void GroundWeather_Select(s16);
+u32 GroundLives_ExecutePlayerScriptActionLives();
+s16 GroundObject_Add(s16 id, GroundObjectData*, s16 group, s8 sector);
+s16 GroundEffect_Add(s16 id, GroundEffectData*, s16 group, s8 sector);
+u32 FlagCalc();
+u32 FlagJudge();
+s32 OtherRandInt(s32); // IT RETURNS s32!! Not s16!
+void FatalError(void* loc, char* fmt, ...) __attribute__((noreturn));
+
+s32 sub_8001658(ScriptUnion832*, s32);
+void sub_8001570(u8*, s16);
+void sub_80015C0(u8*, s16);
+u32 sub_8001784(ScriptUnion832*, s16, u16);
+void sub_80018D8(s32, s32, s32);
+void sub_800199C(u8*, s16, u32, s32);
+u32 sub_8001AB0();
+void sub_8001B88(s16, s32*, s32*);
+u32 ScenarioCalc(s16, s32, s32);
+u32 sub_8001CC4();
+u32 sub_8001D08();
+u32 sub_8001D44();
+void sub_800226C(u8*, s16, s32, u8);
+void sub_80022A0(u8*, s16, s32, u8);
+u32 sub_80022F8();
+u32 sub_8002318();
+u32 sub_80023E4();
+u32 sub_80026CC();
+void sub_80026E8(s16, bool8);
+char sub_8002984(s32, u8);
+u32 sub_8002C60();
+u32 sub_8002D54();
+u8 sub_8002DF0(Position32*, Position32*, Position32*, Position32*);
+s32 sub_8009FB8();
+u32 StopAllMusic_1();
+u32 FadeOutAllMusic();
+u32 xxx_call_start_new_bgm();
+u32 xxx_call_queue_bgm();
+u32 xxx_call_stop_bgm();
+u32 xxx_call_fade_out_bgm();
+u32 xxx_call_play_fanfare_se();
+u32 xxx_call_stop_fanfare_se();
+u32 xxx_call_fade_out_fanfare_se();
+void xxx_call_fade_in_new_bgm(u16 id, u16 arg); //sub_8011900
+bool8 sub_8021700(s32);
+bool8 sub_802FCF0(void);
+
+
+s32 FindItemInInventory();
+u32 ShiftItemsDownFrom();
+u32 sub_80920B8();
```

Comment:
```suggestion
void sub_80920B8(u8 *buffer)
```

From `rescue_team.h`